src/commands/BeautifyCommand.ts

```diff
@@ -0,0 +1,139 @@
+import { BeautifyData } from "unibeautify";
+import * as cosmiconfig from "cosmiconfig";
+import * as fs from "fs";
+
+import {
+  setupUnibeautify,
+} from "../utils";
+import { BaseCommand } from "./BaseCommand";
+
+export class BeautifyCommand extends BaseCommand {
+  protected stdin: NodeJS.ReadStream = process.stdin;
+
+  public beautify(programArgs: IArgs): Promise<string> {
+    const { language, filePath } = programArgs;
+    return setupUnibeautify().then(unibeautify => {
+      if (!language) {
+        const error = new Error("A language is required.");
+        this.writeError(error.message);
+        this.exitCode = 1;
+        return Promise.reject(error);
+      }
+      return Promise.all([
+        this.readConfig(programArgs),
+        this.readText(filePath),
+      ]).then(([config, text]) => {
+        const data: BeautifyData = {
+          filePath: filePath,
+          languageName: language,
+          options: (config as any) || {},
+          text,
+        };
+        return unibeautify
+          .beautify(data)
+          .then((result: string) => {
+            this.writeOut(result);
+            return result;
+          })
+          .catch((error: Error) => {
+            this.writeError(error.message);
+            this.exitCode = 1;
+            return Promise.reject(error);
+          });
+      });
+    });
+  }
+
+  private readConfig(programArgs: IArgs): Promise<any> {
+    const { configFile, configJson } = programArgs;
+    if (configJson) {
+      return this.parseConfig(configJson);
+    } else {
+      return this.configFile(configFile);
+    }
+  }
+
+  private readText(filePath?: string): Promise<string> {
+    // if (this.isTerminal && filePath) {
+    if (filePath) {
+        return this.readFile(filePath);
+    } else {
+      return this.readFromStdin();
+    }
+  }
+
+  private parseConfig(configJson?: string): Promise<object> {
+    if (!configJson) {
```

Comment:
This block shouldn't be needed as it is right now.  `parseConfig` is only called if [`configJson`](https://github.com/Unibeautify/unibeautify-cli/pull/51/files#diff-e1246312359a9883c597b4fc5063af6bR50) is present.  So no check is needed as it's already done.

Reply:
Good 👁 . Fixing now.

Reply:
Fixed.